src/main/java/com/google/plantasy/utils/UserUtils.java

```diff
@@ -54,13 +54,15 @@ public static Entity initializeUser(String userId, String name, String sessionID
         //these values are always empty upon initialization
         int initialScore = 0;
         long initialTime = 0L;
+        long initialUploadPoints = 0L;
         Entity userEntity = new Entity("user");
         userEntity.setProperty("username",name);
         userEntity.setProperty("userID",userId);
         userEntity.setProperty("quiz_timing",initialTime);
         userEntity.setProperty("gameId", "");
         userEntity.setProperty("blobKey", null);
         userEntity.setProperty("score", initialScore);
+        userEntity.setProperty("lastAwardedUploadPoints", initialUploadPoints);
```

Comment:
we should be able to combine lines 57 and 65. It's good for number to be named if we plan on using them anywhere else but, considering the initial points will likely never change or be used anywhere else, we're fine to just hard code 0L on line 65. (this also applies to initial score and initial time.

Reply:
Okay, combined those lines.